### ISSUE TITLE
Make downsample/upsample accept WCS

### ISSUE THREAD

**Issue by astrofrog** [astropy/imageutils#32]

**Comment by josePhoenix** [comment]
I would be interested in this, if this package is still being developed.

**Comment by pllim** [comment]
From the discussions in #44, I don't think this repo is active anymore.

**Comment by astrofrog** [comment]
@josePhoenix - I *think* that downsample/upsample are now in the core package, and are called ``block_replicate`` and ``block_reduce`` - but I think we still need to add the WCS support

**Comment by pllim** [comment]
@mcara implemented some form of basic WCS transformation for shrinking in `scale_image()` function in [stginga.utils](https://github.com/spacetelescope/stginga/blob/master/stginga/utils.py) module, in case you find that interesting.

**Comment by larrybradley** [comment]
Yes, @astrofrog is correct.  I added `block_reduce` and `block_replicate` to `astropy.nddata.utils`, but they do not preserve WCS.  I recall it wasn't so trivial to preserve WCS distortions (but it may be easy now with `gwcs`).

**Comment by larrybradley** [comment]
Also, @pllim is correct.  This repo is dead and should be removed once we migrate any remaining issues to the astropy issue tracker.....
